geopandas/tests/test_plotting.py

```diff
@@ -335,6 +335,45 @@ def test_empty_geometry(self):
         ax = gdf.plot()
         assert len(ax.collections) == 1
 
+    def test_empty_geometry_colors_poly(self):
```

Comment:
Strictly speaking this class is only for point related tests, and there is a separate class for linestring and polygons. Now, we already use this class for generic tests as well. And putting this equivalent test in three different places is also not ideal for maintainability. 
But long term we should maybe think about how to restructure this.

Reply:
True. A bit of restructuring here wouldn't hurt.